docs/pages/guides/how-to/accounts/use-metamask-account.mdx

```diff
@@ -43,29 +43,29 @@ bun install permissionless viem @metamask/delegation-toolkit
 
 ### Create the clients
 
-First we must create the public, (optionally) pimlico paymaster clients that will be used to interact with the Metamask account.
+First we must create the public, (optionally) pimlico paymaster clients that will be used to interact with the MetaMask smart account.
 
 ```ts
 // [!include ~/snippets/accounts/metamask.ts:clients]
 ```
 
 ### Create the signer
 
-Metamask accounts can work with a variety of signing algorithms such as ECDSA, passkeys, and multisig.
+MetaMask Smart Accounts can work with a variety of signing algorithms such as ECDSA, passkeys, and multisig.
 
 For example, to create a signer based on a private key:
 
 ```ts
 // [!include ~/snippets/accounts/metamask.ts:signer]
 ```
 
-### Create the Metamask delegator account
+### Create the MetaMask delegator account
```

Comment:
Should we just use `delegator smart account`? We don't have any guide line for `MetaMask delegator account` and we removed that terminology from our docs as well.